libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -304,6 +309,9 @@ static DNNReturnType load_tf_model(TFModel *tf_model, const char *model_filename
                       &init_op, 1, NULL, tf_model->status);
         if (TF_GetCode(tf_model->status) != TF_OK)
         {
+            TF_DeleteGraph(tf_model->graph);
+            TF_DeleteStatus(tf_model->status);
+            TF_DeleteSession(tf_model->session, tf_model->status);
```

Comment:
shall we delete status again here?

Reply:
Sorry, i did get it, delete again?
If it's not TF_OK here, it should be it's first time to call TF_DeleteStatus(tf_model->status).

Reply:
will tf_model->status be assigned/allocated again in the function TF_DeleteSession?

Reply:
OK, i get it, actually TF_DeleteSession(..) will not assign tf_model->status again. But i still move this 2 lines ahead, which makes it more resonable.